dockerfiles/init/modules/keycloak/files/che/login/messages/messages_en.properties

```diff
@@ -0,0 +1,240 @@
+product=Eclipse Che
```

Comment:
I'm not asking to replace "our" messages with "upstream" messages.
but if you upgrade keycloak later, you may have new constants to update/change.
And how do we know what was the "customized" strings and the "upstream" strings.

Maybe that you've only changed product name and that's all but here we don't know what were the full changes.
By knowing the changes, then, it's easier to upgrade keycloak